build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -2,10 +2,20 @@
   xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <PropertyGroup>
+    <XUnitPath>$(NuGetPackageRoot)\xunit.runner.console\2.1.0\tools\xunit.console.x86.exe</XUnitPath>
```

Comment:
As poorly as we always have done. Observe the path was already in this file, so this is nothing new. I'm open to ideas if you have them.